BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -483,12 +585,34 @@ def gen_container_bin (container_list, out_dir, inp_dir, key_dir = '.', tool_dir
 		out_file = container.create (each)
 		print ("Container '%s' was created successfully at:  \n  %s" % (container.header.signature.decode(), out_file))
 
+def gen_layout (comp_list):
+	# prepare the layout from individual components from '-cl'
+	layout 		= "('PODS', 'PodsImage.bin', 'CLASSIC', 'RSA2048' , 'TestSigningPrivateKey.pem', 0x10, 0),\n"
```

Comment:
shouldn't the key be passed in through command line in the similar way as IasTool ?

Reply:
add the same for the final  output file name.  It should be from command line.

Reply:
I think when '-cl' is provided,  you also need to provide  key path and output file path.

Reply:
Addressed these in new patch.